modules/commons/src/main/java/org/apache/synapse/commons/json/JsonUtil.java

```diff
@@ -468,6 +534,35 @@ private static void removeIndentations(OMElement elem) {
         }
     }
 
+    /**
+     * Remove indentations of a OMElement.
+     * If PRESERVE_SPACES property in the requiredProperties is true, spaces inside XML elements will be preserved.
+     *
+     * @param requiredProperties Required properties
+     * @param elem               OMElement needed to remove indentations
+     */
+    private static void removeIndentations(Map requiredProperties, OMElement elem) {
```

Comment:
"required" part of "requiredProperties" property have no meaning let's rename as just "properties"

Reply:
Done